rts/Lua/LuaConstEngine.cpp

```diff
@@ -61,5 +64,14 @@ bool LuaConstEngine::PushEntries(lua_State* L)
 		LuaPushNamedNumber(L, "maxPiecesPerModel", MAX_PIECES_PER_MODEL);
 	lua_rawset(L, -3);
 
+	// Fixes issue #1731.
+	LuaPushNamedNumber(L, "gameSpeed"      , GAME_SPEED);
+
+	lua_pushliteral(L, "textColorCodes");
+	lua_createtable(L, 0, 3);
+		LuaPushNamedChar(L, "Color"          , CglFont::ColorCodeIndicator  );
```

Comment:
Other returned tables in the project use camelCase rather than PascalCase. This appears to be breaking that convention. PascalCase is typically reserved for globals in Lua.

(Maybe get a second opinion from @sprunk or @badosu on this in case I'm mistaken.)

Reply:
Some examples: https://github.com/beyond-all-reason/spring/blob/master/rts/Lua/library/Types.lua#L50-L110

Reply:
It should stay as-is because it's a copy of https://github.com/beyond-all-reason/spring/blob/198c84e8b3c2b57490d790dddcb7efd69b11c025/rts/Lua/LuaConstGame.cpp#L327-L332